app/views/content_items/detailed_guide.html.erb

```diff
@@ -7,9 +7,7 @@
     <%= render 'govuk_component/title', @content_item.title_and_context %>
   </div>
   <div class="column-third">
-    <%= render 'shared/available_languages',
-      translations: @content_item.available_translations
-    %>
+    <%= render 'components/translation-nav', translations: @content_item.available_translations %>
```

Comment:
If `available_translations` returned `{ translations: … }` this could be simplified to:
```erb
<%= render 'components/translation-nav', translations: @content_item.available_translations %>
```

This might make it easy to refactor the API in the future.